_posts/0000-01-01-intro.md

```diff
@@ -3,4 +3,4 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
```

Comment:
Replace line 5 of your file with a quotation or meme and witty caption

As Kanye West said:

> We're living the future so
> the present is our past.

What a thinker!